### ISSUE TITLE
[feature request] Show remaining printtime on locked screen

### ISSUE THREAD

**Issue by tscharly** [X1Plus/X1Plus#121]
We locked the printer @ our spaceship and people are complaining that they cannot see the remaining printing time.

**Comment by jwise** [comment]
Yeah, good idea.  Adding stuff to the lock screen is easy.  Having a good config UI for it is hard.  Do you have ideas of what in general you'd like to see on the lock screen?

**Comment by tscharly** [comment]
-remaining print time
-editable Text
-IP

**Comment by john-f-chamberlain** [comment]
- Picture of the model being printed would be cool

**Comment by tscharly** [comment]
-actual printing speed

**Comment by tscharly** [comment]
Stop button on lockscreen

**Comment by psiberfunk** [comment]
> Stop button on lockscreen

Uhm … no ? That defeats the purpose

**Comment by tscharly** [comment]
> > Stop button on lockscreen
> 
> Uhm … no ? That defeats the purpose of a lock screen .. which is to prevent people from making alterations to the status of the system

Also prevents people from stoping printer if shit is happening. Maybe opt in/opt out for stop-button. 

Next idea:
- show layer height on lockscreen

**Comment by psiberfunk** [comment]
> > > Stop button on lockscreen
> > 
> > 
> > Uhm … no ? That defeats the purpose of a lock screen .. which is to prevent people from making alterations to the status of the system
> 
> Also prevents club members from stoping printer if shit is happening. Maybe opt in/opt out for stop-button.
> 
> Next idea:
> 
>     * show layer height on lockscreen

That's what the power button at the back is for I guess ?  I mean, if you don't want messing with stuff, that's what the lockscreen is for.. if you DO want them to be able to stop things.. then don't have a lock screen set?
